graphium/trainer/predictor.py

```diff
@@ -39,6 +39,7 @@ def __init__(
         metrics_on_training_set: Optional[Dict[str, List[str]]] = None,
         flag_kwargs: Dict[str, Any] = None,
         task_norms: Optional[Dict[Callable, Any]] = None,
+        metrics_every_n_steps: Optional[bool] = 1,
```

Comment:
Especially for smaller models, computing stats and logging is huge bottleneck. This parameter lets the user trade speed for detailed stats at each batch.